airviewer.py

```diff
@@ -41,23 +41,30 @@
 import matplotlib.animation as animation
 import matplotlib.ticker as plticker
 
+requests.packages.urllib3.util.ssl_.DEFAULT_CIPHERS += 'HIGH:!DH:!aNULL'
+try:
+    requests.packages.urllib3.contrib.pyopenssl.DEFAULT_SSL_CIPHER_LIST += 'HIGH:!DH:!aNULL'
+except AttributeError:
+    # no pyopenssl support used / needed / available
+    pass
+
 USERNAME = 'ubnt'
 PASSWORD = 'ubnt'
 HOST = "192.168.1.20"
 PORT = 18888
 TIMEOUT = 10
 FRAME_SPEED = 1
 
-LOGIN_URI = 'http://' + HOST + ':80/login.cgi'
-#LOGIN_URI = 'https://' + HOST + ':443/login.cgi'
+#LOGIN_URI = 'http://' + HOST + ':80/login.cgi'
+LOGIN_URI = 'https://' + HOST + ':443/login.cgi'
```

Comment:
Could making HTTPS default be a seperate commit? Most preferable with a option to fall-back to legacy HTTP login if needed.

Reply:
Indeed, sorry about that ;-)